index.html

```diff
@@ -10,125 +10,143 @@
   <body>
     <header>
       <a id="logo">Farida's Logo</a>
+      <nav id="top">
+        <a>Portfolio</a>
+        <a>About</a>
+        <a>Contact</a>
+      </nav> 
       <nav>
         <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
       </nav>
```

Comment:
- [ ] [Optional] According to your requirements and your Figma template, in the desktop version you have to hide the 'hamburguer' icon